derivation.nix

```diff
@@ -112,18 +169,21 @@ in
                 inherit nativeBuildInputs;
 
                 unpackPhase = concatStringsSep "\n"
-                  (
+                  ( [ # components is an empty list for non workspace builds
```

Comment:
The comment would need to change here, but `mkdir -p .` is a no-op, so having a non-empty components list wouldn't hurt in this case.

Reply:
Comment stays, as components is empty list for non-workspace build.